routes/products.js

```diff
@@ -1,3 +1,42 @@
 const productsRouter = require("express").Router();
+const { getProductById, getAllProducts, createProduct, getProductByName } = require("../db")
 
-productsRouter.get("/", async )
\ No newline at end of file
+productsRouter.get("/", async (req, res, next) =>{
+    try {
+        const allProducts = await getAllProducts();
+        res.send(allProducts)
+    } catch (error) {
+        throw error
+    }
+} );
+
+productsRouter.get("/:productId", async (req, res, next) => {
+    try {
+        const products = await getProductById(id);
```

Comment:
would it make more sense to call this `product` since it's just one product?

Reply:
Edits were implemented.